packages/terra-form-textarea/src/TextareaField.jsx

```diff
@@ -181,12 +208,16 @@ const TextareaField = (props) => {
         {...inputAttrs}
         disabled={inputAttrs.disabled || disabled}
         id={inputId}
+        isAutoResizable={isAutoResizable}
```

Comment:
We might need to back patch each of these similar to `disabled={inputAttrs.disabled || disabled}` above to remain passive. Otherwise anyone previously using `inputAttrs` to set these values would be overwritten.

A quick test would be to create an example using inputAttrs and see if the values are overwritten:

```jsx
// Does this still work after these changes?
<TextareaField inputAttrs={{ isAutoResizable: true }} />
```

Reply:
Changed for all new props ca966a9